server/Resources/Resources.resx

```diff
@@ -3548,7 +3548,10 @@ Set to "External URL" to use an API definition that is hosted elsewhere.</value>
     <value>Validating...</value>
   </data>
   <data name="vstsReleaseBuildPermissions" xml:space="preserve">
-    <value>You do not have permissions to create a Build pipeline or a Release pipeline on this project. Contact your project administrator</value>
+    <value>You do not have permissions to create a Build pipeline or a Release pipeline on this project. Contact your project administrator or choose a different project</value>
+  </data>
+  <data name="vstsAgentQueuePermissions" xml:space="preserve">
+    <value>You do not have permissions to use at least one of the available agent pools in this project . Contact your project administrator or choose a different project</value>
```

Comment:
The message can be rephrased to say, You do not have permission to use any of the available agent pools in …..